staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -0,0 +1,253 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apiserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sort"
+
+	"github.com/emicklei/go-restful"
+	"github.com/go-openapi/spec"
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/kube-openapi/pkg/aggregator"
+	"k8s.io/kube-openapi/pkg/builder"
+	"k8s.io/kube-openapi/pkg/common"
+	"k8s.io/kube-openapi/pkg/handler"
+
+	"k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/kube-aggregator/pkg/apis/apiregistration"
+)
+
+const (
+	aggregatorUser = "system:aggregator"
+)
+
+type openAPIAggregator struct {
+	// Map of API Services' OpenAPI specs by their name
+	openAPISpecs map[string]*openAPISpecInfo
+
+	// provided for dynamic OpenAPI spec
+	openAPIService *handler.OpenAPIService
+
+	// Aggregator's OpenAPI spec.
+	aggregatorOpenAPISpec *spec.Swagger
+
+	// Local delegate's OpenAPI spec.
+	localDelegatesOpenAPISpec *spec.Swagger
```

Comment:
Renamed it to `inProcess...`. I like is as much as the previous one (which is not much :D)